WordPress/UITestsFoundation/Screens/Me/SupportScreen.swift

```diff
@@ -27,6 +27,21 @@ public class SupportScreen: ScreenObject {
         )
     }
 
+    public func contactSupport() throws -> ContactUsScreen {
+        app.cells["contact-support-button"].tap()
+        addContactInformationIfNeeded()
+        return try ContactUsScreen()
+    }
+
+    private func addContactInformationIfNeeded() {
+        let emailTextField = app.textFields["Email"]
```

Comment:
Would it be possible to group this locator here for reusability? https://github.com/wordpress-mobile/WordPress-iOS/blob/32983737f021caf2957c07f6c302aabe8fe2abbd/WordPress/UITestsFoundation/Screens/Me/SupportScreen.swift#L12

Reply:
Good question! I'll just share my thoughts here, please don't take it as resistance for changing.

This element is in a dialog that kicks in when you tap `Contact Support` or `My Tickets` without having an email set in `Contact Email`.

![image](https://user-images.githubusercontent.com/42008628/143026570-5920bde4-1a63-4f29-889e-eced1631cf2c.png)

My initial thought was that this element is not actually part of the SupportScreen and, as we are using Screen Objects model, having that element along with other SupportScreen elements might be confusing. I also thought it might be too much to have a specific ScreenObject for this dialog.

Let me know if it makes sense to you or I'll just change its. =)

Reply:
Fair enough 😄 

I would have grouped it together as part of the `SupportScreen` if it is an element that is only going to be displayed when on that screen. That said, I don't believe we have agreed to the style and standards of writing tests yet, so should be good to keep as it (your explanation does make sense) and we can talk about this later so that expectations are more aligned.

Reply:
> I would have grouped it together as part of the SupportScreen if it is an element that is only going to be displayed when on that screen.
Yeah, it makes total sense to me.

> I don't believe we have agreed to the style and standards of writing tests yet, so should be good to keep as it 
I don't know if there was any agreement in the past but the two apps I've worked so far have completely different standards. We definitely need to approach it holistically.

Reply:
@jostnes as I did address the locators as you suggested. Thank you for pointing that out. =)